### PR TITLE
Math-Magicians: Components

### PR DESCRIPTION
**Pull request by kessie2862**
In this PR, I

- [ ] Deleted all boilerplate `files/codes`
- [ ] Created and styled `calculator` component
- [ ] Fixed linter errors

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+function Calculator() {
+  return (
+    <div className="container">
+      <div className="btns">
+        <button className="btn" type="button" value="AC">
+          AC
+        </button>
+        <button className="btn" type="button" value="+/-">
+          +/-
```
Comment (rloterh) [inline src/components/Calculator.js]: - [ ] In this activity we want you to use one child (custom) component inside the _Calculator_ component. Please improve upon this aspect.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Prosper, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!
🏁 🚦 ⛪ 🚜 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Good job creating and adding components to the `component` directory!

## Required Changes ♻️
- [ ] MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it.
- [ ]  The _Author's Explanation_ Zoom video link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

_Kindly check the other inline comment(s) under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (kessie2862) [reply src/components/Calculator.js]: Can you please elaborate?

Comment (rloterh) [reply src/components/Calculator.js]: Hi @kessie2862 

I think you can create a Button component which will hold all the buttons of your app. And then, import and use the Button component you've created inside the Calculator component.  Doing so will keep the component decoupled and organized.

Comment (kessie2862) [reply src/components/Calculator.js]: Alright. Thanks

- [ ] By the way, the MIT LICENSE in my README works. It's not broken as you indicated. 
- [ ] Also, I did not even include  _Author's Explanation Zoom video link_ so I don't understand why you said the link was broken.

Comment (rloterh) [reply src/components/Calculator.js]: Hi @kessie2862 

Kindly check carefully in your ReadMe:
image


[screencast-www.google.com-2023.06.06-18_35_28.webm](https://github.com/kessie2862/Math-Magicians/assets/12745474/b7d0d27f-7ebd-4821-85dd-1f02f405a67f)

Comment (kessie2862) [reply src/components/Calculator.js]: That might be yesterday. I've already fixed all that. 

readme

**Review by aamirkhan2478 — APPROVED** [review]
Hi @kessie2862 , 

GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ You used Gitflow correctly :+1: 
✅ No linter error :+1: 
✅ Good commit messages :+1: 
✅ Good descriptive PR :+1: 
✅ Well done :clap: you did all changes given by the previous reviewer :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**